stdlib/src/builtin/object.mojo

```diff
@@ -107,20 +120,11 @@ struct _RefCountedListRef:
 
     @always_inline
     fn copy(self) -> Self:
-        _ = self.lst.bitcast[_RefCountedList]()[].refcount.fetch_add(1)
+        _ = self.lst.bitcast[_RefCountedList]()[].impl
```

Comment:
I think this trigger `__init__` which do `increment`

Reply:
If I understand correctly, you're saying this will invoke `_RefCountedList.__init__` which then calls the `Arc` constructor which then eventually bumps the inner data refcount by `1`, preserving the same behavior as before.